CONTRIBUTING.adoc

```diff
@@ -610,3 +610,18 @@ The "details" link for the task goes to a page for the entire action and you wil
 
 If no GitHub statuses are shown, this might indicate that the invocation itself has failed.
 Check the link:https://cicero.ci.iog.io/action/53f25880-8fde-4e24-8ce7-6b7568777e00[plutus action page] for details, including logs and the full GitHub event payload.
+
+== Project roles and responsibilities
+
+- The regular contributors to the Haskell code, all of whom can review and merge PRs are:
+    - @michaelpj
+    - @effectfully
+    - @kwxm
+    - @bezirg
+    - @thealmarty
+    - @zliu41
+- The maintainer of the documentation is @joseph-fajen.
+- The maintainer of the Agda code is @jmchapman.
```

Comment:
this may change soon?

Reply:
Yeah, I wasn't sure whether it was more accurate to say this or to say that it's unmaintained. I think in practice this is still accurate: if we had a big problem with it we'd go running for James!

Reply:
also there's a new person?

Reply:
yeah, but they won't be doing much for a bit. we can change this!